docs/accept-a-maximum-number-of-orders-per-day/README.md

```diff
@@ -3,14 +3,15 @@
 * [task.json](../../tasks/accept-a-maximum-number-of-orders-per-day.json) (for import/export)
 * [Task script](./script.liquid)
 
-This task works by monitoring the number of orders created per day, and clearing the inventory for all in-stock items when the daily order limit is reached. Optionally, this task can restore inventory to its original levels at midnight the next day, or on demand.
+This task works by setting your inventory to zero when the order limit is reached. (Specifically, this means setting inventory levels to 0 for all items that have a greater-than-zero inventory level.) There are no popups, or any specific messaging - your inventory will simply be dropped to zero, and if your shop is configured to stop selling out-of-stock products, your customers will be prevented from making additional purchases.
```

Comment:
:D So here's what's happening: the first paragraph of the docs is going to be used as the task summary, seen on [usemechanic.com](https://usemechanic.com) and elsewhere. (I [updated CONTRIBUTING.md](b605e01) to document this - my bad for forgetting that detail earlier.)

Bearing that in mind, please update the task docs, with a summary-friendly first paragraph, and any other details following. Specifically make sure to address the fact that merchants may now configure some product-level messaging, if they like!

Reply:
Took another stab at this, but in trying to follow how this task doc was originally segmented, I'm not sure I met your stated goal of separating summary-friendly from the nitty gritty. Happy to adjust further if needed.

Reply:
👍 👍  It's good enough to merge! I'll follow up with some tweaks in another commit. :) Thanks!